### ISSUE TITLE
Script execution fails with no module

### ISSUE THREAD

**Issue by zauberparacelsus** [MyWorldLLC/Chipmunk#31]
When a script contains no module, execution fails.

```
from chipmunk.system import println

def main() {
    println("Hello World!")
}
```

The script above returns the error `java.lang.ClassFormatError: Illegal class name "" in class file`

**Comment by danielperano** [comment]
This is a bug introduced by the JVM port - with no `module` statement, Chipmunk places the code into the "default" module, which in the legacy ChipmunkVM was a module with the empty string for a name. This is incompatible with the JVM bytecode produced for the module, however, so this needs to change to an easily identifiable, non-empty name for the default module.

**Comment by danielperano** [comment]
Fixed by 3dfe737.
